src/check_clickhouse.py

```diff
@@ -320,17 +320,20 @@ def _optimized_request(self, table: str, local_table: list) -> str:
         """
         This method is trying to optimize request by adding 'WHERE' clause
         with partitioning key. In the best case WHERE won't match anything
-        and SELECT won't return anything. Here the next steps are done:
+        and SELECT will over almost immediately. Here the next steps are done:
```

Comment:
'SELECT will over almost immediately' .. there is something wrong with the sentence. Not sure, what you meant, maybe:
```suggestion
        and SELECT will be over almost everything. Here the next steps are done:
```
?